### ISSUE TITLE
Implement a tag function from local file system

### ISSUE THREAD

**Issue by faiq** [Clarifai/clarifai-go#3]
according to the documentation [here] (https://developer.clarifai.com/docs/tag) you are able to make requests to the tag parameter with a local file, but this functionallity is not availble in this [client] (https://github.com/Clarifai/clarifai-go/blob/master/requests.go#L93). it seems to be implemented in the python client [here] (https://github.com/Clarifai/clarifai-python/blob/master/clarifai/client/client.py#L189). i'll be willing to put in a pull request.

**Comment by samuelcouch** [comment]
You are correct, that wasn't implemented. A PR would be much appreciated! :+1:

**Comment by zquestz** [comment]
This is merged into my fork: https://github.com/zquestz/clarifai-go
